src/components/LogIn/LogIn.js

```diff
@@ -8,29 +8,47 @@ import slackLogo from "../../assets/slack-logo.png";
 function LogIn() {
   const [username, setUsername] = useState("");
   const [pw, setPw] = useState("");
-  const [header, setHeader] = useContext(APIHeaders);
-  //modal
+
   const [showModal, setShowModal] = useState(false);
   const openModal = () => {
     setShowModal((prev) => !prev);
   };
 
-  useEffect(() => {
-    if (header["access-token"] == null) return;
-    setTimeout(() => {
-      getAllUsers(header);
-    }, 200);
-  }, [header]);
+
   const getAllUsers = (header) => {
     axios({
       method: "get",
       url: "http://206.189.91.54//api/v1/users",
       headers: header,
       redirect: "follow",
     })
-      .then((res) => console.log(`Success: ${res}`))
+      .then((res) => {
+        console.log(`Success: ${res}`);
+      })
       .catch((e) => console.log(`Error: ${e}`));
   };
+
+  const handleEmailChange = (e) => {
+    setUsername(e.target.value)
+  }
+
+  const handlePasswordChange = (e) => {
+    setPw(e.target.value)
+  }
+
+  const handleHeader = (res) => {
+
+    if (res.data) {
+      localStorage.setItem("access-token", JSON.stringify(res.headers["access-token"]));

```

Comment:
local storage is an intuitive way of accessing without using context! nice

Reply:
But I think you can remove JSON.stringify for this one because we are using axios.